src/Elasticsearch.Net/ConnectionPool/SniffingSortedStickyConnectionPool.cs

```diff
@@ -5,74 +5,31 @@
 
 namespace Elasticsearch.Net
 {
-	public class SniffingSortedStickyConnectionPool : IConnectionPool
+	public class SniffingSortedStickyConnectionPool : SniffingConnectionPool
```

Comment:
I've just pushed a change through doing that, and also making `StickyConnectionPool` inherit from `StaticConnectionPool`, since there didn't seem to be any reason to keep them separate any more that I could see. Hope that's ok!